srcpkgs/goldendict-ng/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'goldendict-ng'
+pkgname=goldendict-ng
+version=24.05.05
+revision=1
+_version_name=LiXia.ecd1138c
+build_style=cmake
+configure_args="-DUSE_ALTERNATIVE_NAME=ON"
+makedepends="qt6-base qt6-speech-devel qt6-svg-devel qt6-qt5compat-devel qt6-base-devel
+ qt6-webengine-devel qt6-multimedia-devel qt6-speech-plugin-flite
+ qt6-speech-plugin-mock qt6-speech-plugin-speechd pkg-config ffmpeg-devel
```

Comment:
pkg-config should be hostmakedepends, probably also need qt6-tools in hostmakedepends